source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -766,10 +801,6 @@ socket:
   delete the corrupted bitmaps manually. This will fail HA VM flows and
   storage operations. (libvirt/qemu).
 
-- When creating a snapshot, an active bitmap should be copied to the new
-  top layer, and qemu should continue to track changes, writing into the
-  copied bitmap (libvirt).
-
```

Comment:
This change is not related, please send another patch for this.

Reply:
ack